algoliasearch/algoliasearch.go

```diff
@@ -212,6 +212,14 @@ type Client interface {
 	// also accepts extra RequestOptions.
 	MultipleQueriesWithRequestOptions(queries []IndexedQuery, strategy string, opts *RequestOptions) (res []MultipleQueryRes, err error)
 
+	// MultipleGetObjects retrieves objects, based on their objectID, from
+	// multiple indices in a single call.
+	MultipleGetObjects(requests []IndexedGetObject) (res MultipleGetObjectsRes, err error)
```

Comment:
Changing the contract breaks BC, I guess we are releasing this in a major release?

Reply:
Yes indeed but we are actually breaking the compatibility for a long time for every new feature because of this huge top interface. We had no choice for this v2 but we will make sure to not reproduce the issue in the coming v3 (probably with interface segregation). So far, this issue impacts the customers using the interface to mock the client.

Reply:
Sounds good!